contrib/python/examples/eg_image_list.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+"""Example: Show all images on system."""
+
+import podman
+
+print('{}\n'.format(__doc__))
+
+
+with podman.Client() as client:
+    # Note: not recommended on a system with a large number of images
+    list(map(print, client.images.list()))
```

Comment:
Really not good on a bigger system, but is there a way to sort the list by image name or what have you?

Reply:
I rewrote this example to be more cross-language friendly. :)  I don't want to get into sorting in this example to keep it dirt simple.  eg_containers_by_image.py gives an example of sorting and grouping.

Reply:
That's fine, I'm fine with this example as is.  I should have framed my question better, I was more asking if there was a sorting example somewhere and there sounds like there is so :+1: